README.md

```diff
@@ -1395,7 +1395,7 @@ Shorthand requests and manual `z.request([url], options)` calls support the foll
 * `method`: HTTP method, default is `GET`.
 * `headers`: request headers object, format `{'header-key': 'header-value'}`.
 * `params`: URL query params object, format `{'query-key': 'query-value'}`.
-* `body`: request body, can be a string, buffer, or readable stream. Default is `null`.
+* `body`: request body, can be a string, buffer, readable stream or plain object. When it is an object/array and the `Content-Type` header is `application/x-www-form-urlencoded` the body will be transformed to a query string, otherwise we'll set the header to `application/json; charset=utf-8` and JSON encode the body. Default is `null`.
```

Comment:
You might want to update the example below too (line 1416):

```
   // only provide body, json or form...
   body: '{"hello":"world"}', // or 'hello=world'
```

> the body will be transformed to a query string

This may be misinterpreted as meaning the URL of the request will be modified to include the body as the query string.